src/ert/gui/simulation/experiment_panel.py

```diff
@@ -142,23 +150,24 @@ def addExperimentConfigPanel(self, panel, mode_enabled: bool) -> None:
 
         if not mode_enabled:
             item_count = self._experiment_type_combo.count() - 1
-            sim_item = self._experiment_type_combo.model().item(item_count)
+            combo_model = self._experiment_type_combo.model()
+            sim_item = self._experiment_type_combo.model().item(item_count)  # type: ignore
```

Comment:
You maybe intended:
```
sim_item = combo_model.item(item_count)
```

Reply:
Ended up removing `combo_model` as the problem is instead that `model()` returns `QAbstractItemModel` rather than `QComboBoxModel`